diff -ru xfce4-xkb-plugin-0.5.3.3.orig/panel-plugin/xkb-config.c xfce4-xkb-plugin-0.5.3.3/panel-plugin/xkb-config.c
--- xfce4-xkb-plugin-0.5.3.3.orig/panel-plugin/xkb-config.c	2010-01-23 15:16:47.000000000 +0000
+++ xfce4-xkb-plugin-0.5.3.3/panel-plugin/xkb-config.c	2010-01-23 15:17:10.000000000 +0000
@@ -214,7 +214,7 @@
 
     gdk_window_remove_filter (NULL, (GdkFilterFunc) handle_xevent, NULL);
 
-    xkl_engine_stop_listen (config->engine);
+    xkl_engine_stop_listen (config->engine, XKLL_TRACK_KEYBOARD_STATE);
 }
 
 gint
