--- xfce4-xkb-plugin-0.8.1/configure.ac~	2017-09-08 22:51:33.000000000 +0200
+++ xfce4-xkb-plugin-0.8.1/configure.ac	2019-08-25 21:56:39.782881196 +0200
@@ -81,7 +81,7 @@
 XDT_CHECK_PACKAGE([LIBXFCE4UTIL], [libxfce4util-1.0], [4.12.0])
 XDT_CHECK_PACKAGE([LIBXFCE4UI], [libxfce4ui-2], [4.12.0])
 XDT_CHECK_PACKAGE([XFCONF], [libxfconf-0], [4.12.1])
-XDT_CHECK_PACKAGE([LIBXKLAVIER], [libxklavier], [5.4])
+XDT_CHECK_PACKAGE([LIBXKLAVIER], [libxklavier], [5.3])
 XDT_CHECK_PACKAGE([LIBRSVG], [librsvg-2.0], [2.40])
 XDT_CHECK_PACKAGE([LIBWNCK], [libwnck-3.0], [3.14])
 XDT_CHECK_PACKAGE([GARCON], [garcon-1], [0.4.0])
